### ISSUE TITLE
Termite crashes trying to enter selection mode

### ISSUE THREAD

**Issue by javiertury** [thestinger/termite#565]
I built termite using the latest termite and vte-ng source code. After pressing CTRL-SHIFT-<space> termite crashes with the following error

    termite: symbol lookup error: termite: undefined symbol: vte_terminal_disconnect_pty_read

**Comment by jelly** [comment]
I can't reproduce this issue. did you switch the vte-ng source branch to 0.50.2-ng?

**Comment by javiertury** [comment]
I was on branch 0.48.2-ng but the same problem happens with vte-ng 0.50.2-ng branch. After digging a bit, I found the problem. Even though at compile time I explicitly tell termite where to look for the vte-ng library and headers using `PKG_CONFIG_PATH`, it ignores the vte-ng library path at runtime and instead uses the regular vte library. 

    PKG_CONFIG_PATH=/usr/local/lib/pkgconfig/ make install

Instead of executing termite directly, I have to do this:

    LD_LIBRARY_PATH=/usr/local/lib/ termite

Right now I'm trying to find better long term solutions, but I am stalled. I can't delete the regular vte library since many gnome and virtualization packages depend on it. Then I though, maybe if I create a patch for termite allowing to pass flags to pkg-config I can solve it by using the static flag. I've tried to statically link termite dependencies modifying the Makefile `shell pkg-config --static ...` but it didn't work. The reason is that `vte-2.91.pc` file has no support for static linking.
